nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -108,8 +108,9 @@ def process(self, context, node):
         if latest_graph != latest_stored_graph:
             # Since a new survey is performed once every two hours, it should be impossible to insert new data more
             # often than once every hour.
-            if latest_stored_graph:
-                assert timezone.now() - streams[0]['latest_datapoint'] > datetime.timedelta(hours=1)
+            assert not latest_stored_graph or timezone.now() - streams[0]['latest_datapoint'] >= datetime.timedelta(
+                hours=1,
+            )
```

Comment:
120 is not a strict limit. In this case for example I would not do it myself. :-)